mitmproxy/addons/script.py

```diff
@@ -142,7 +148,10 @@ def __init__(self):
     def load(self, loader):
         loader.add_option(
             "scripts", typing.Sequence[str], [],
-            "Execute a script."
+            """
+                Execute a script. The script name may include wild card. If you include wild card,
+                don't forget to enclose the script name in single or double quotes. Example: mitmproxy -s "some/folder/*.py".
```

Comment:
We should explain here what's the purpose of having a wildcard, the fact that it needs to be quoted depends on your console. How about something like this?
```
                Execute a script. 

                Experimental: If the script name includes a wild card, all matching scripts are executed.
                Addons created after startup are not picked up.
                Example: mitmproxy -s "some/folder/*.py".
```

Reply:
It looks great.
I will apply this.